docs/source/common/progress_bar.rst

```diff
@@ -0,0 +1,138 @@
+.. testsetup:: *
+
+    from pytorch_lightning.trainer.trainer import Trainer
+
+Progress Bar
+============
+
+Lightning supports two different types of progress bars (`tqdm <https://github.com/tqdm/tqdm>`_ and `rich <https://github.com/Textualize/rich>`_). :class:`~pytorch_lightning.callbacks.TQDMProgressBar` is used by default,
```

Comment:
1. should this mention the base progress bar class somewhere? 

or at this point, do we even need the base class? 
- progress tracking is done in the loop
- once https://github.com/PyTorchLightning/pytorch-lightning/issues/10982 is in, then enable/disable aren't required on the progress bar API

2. should we mention that a progress bar is automatically enabled with the Trainer, and to disable it, one should do this:`Trainer(enable_progress_bar=False)`

Reply:
Updated!